tests/jest/api/RestApi.spec.js

```diff
@@ -1,19 +1,675 @@
-import RestApiTestUtils from '@/lib/test/RestApiTestUtils';
+import {
+  AuthScope,
+  CentrelineType,
+  HttpStatus,
+  StudyHours,
+  StudyRequestReason,
+  StudyRequestStatus,
+  StudyType,
+} from '@/lib/Constants';
+import BackendClient from '@/lib/api/BackendClient';
+import UserDAO from '@/lib/db/UserDAO';
+import AuthState from '@/lib/model/AuthState';
+import DAOTestUtils from '@/lib/test/DAOTestUtils';
+import { generateUser } from '@/lib/test/random/UserGenerator';
+import DateTime from '@/lib/time/DateTime';
+import { initialize } from '@/web/MoveServer';
 
-beforeAll(RestApiTestUtils.startup, RestApiTestUtils.TIMEOUT);
-afterAll(RestApiTestUtils.shutdown, RestApiTestUtils.TIMEOUT);
+class InjectBackendClient {
```

Comment:
we can leave this here for now, but we should eventually move this to its own file in `lib/test` and add tests.